packages/iov-bns/src/types.ts

```diff
@@ -99,6 +99,11 @@ export enum VoteOption {
   Abstain,
 }
 
+export enum ActionKind {
```

Comment:
Please add explicit string raw values to thse cases, since all transaction data needs to go over JSON interfaces and source and destination may use different versions of IOV-Core, so this must be extensible in a way that the same raw value never translates to a different enum case.